src/main/java/com/spotify/docker/client/messages/swarm/Service.java

```diff
@@ -71,6 +74,12 @@
     @Nullable
     public abstract String serviceName();
 
+    /**
+     * Filter by label.
+     */
+    @Nullable
+    public abstract ImmutableMap<String, String> labels();
```

Comment:
this contains the labels associated with the Service, right? That were created when the service was created? i.e. this isn't the labels used in a "list services" filter